000_introduction/000_introduction.tex

```diff
@@ -1,8 +1,14 @@
+
+This project proposes a twofold bifurcation from conventional CPR problems: firstly, there are no provisions made to the common pool, meaning that we cannot refer to this problem as a linear public goods (LPG) game. Instead, players must maintain a collective resource without offering replenishment from their personal pool. This effectively constructs a scenario where all players make appropriation with insufficient provision, yielding a system of \textit{N free-riders}. 
+
+Secondly, Ostrom notes that common-pool management problems are provably solvable through the introduction of self governing institutions \cite{ostromCollectiveAction}, offering a contradiction to the tragedy of the commons and zero contribution thesis \cite{zeroCon}. This project instead offers a scenario without centralised governance and an inability to monitor other players, effectively preventing this research from complying with Ostrom’s ``design principles for enduring self-governing institutions.''
+
+The absence of a centralised authority, coupled with an inability to form self-governing institutions proposes a seemingly unsolvable problem: a common pool of resources must be maintained across a network of players, each acting rationally to maximise local utility \cite{oberUtil}, without the prospect of forming enforceable rules. This necessitates the presence of a form of self-organisation sufficiently powerful as to overcome the challenges set by the nature of this problem, whilst accepting the risk that such a self-organising mechanism may act perniciously under these circumstances \cite{dobSteg}.
+
+Treaties act as a form of institutionalised power between players, serving both ``as a juristic act and as a rule'' \cite{reuter1995introduction}, with society coming to a mutual agreement upon the importance of such a legal device. Overall, it is the topic of this research to evaluate if the use of treaties as a means of self-organisation can not only combat the difficulties poised by such an unconventional CPR game, but achieve a level of stability if this game is played iteratively in an \textit{N player} scenario.
 
 \section{Our Goals}
```

Comment:
Do we need to add some goals? If yes, we could add very high-level goals such as:

- build a piece of software capable of simulating the given scenario
- design agents that interact with their environment and with each other, to eventually form a self-organised multi-agent system
- using the infrastructure and the proposed agents, perform experiments using different configuration, and analyse the results
- maybe list a few theoretical question we would like to answer, for example "Can this system reach a form of stability by using treaties?", etc